### PR TITLE
Add a new attribute export for PimGento

### PR DESCRIPTION
**Pull request by damien-carcel**

### CODE REVIEW

File: Processor/AttributeToFlatArrayProcessor.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+/**
+ * Attribute to flat array processor.
+ *
+ * @author    Damien Carcel <damien.carcel@akeneo.com>
+ * @copyright 2015 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class AttributeToFlatArrayProcessor extends AbstractProcessor implements ItemProcessorInterface
+{
+    const ITEM_SEPARATOR = ',';
```
Comment (fitn) [inline Processor/AttributeToFlatArrayProcessor.php]: staticvar

File: Processor/AbstractProcessor.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigurationFields()
+    {
+        return [];
+    }
+}
```
Comment (fitn) [inline Processor/AbstractProcessor.php]: Imho, it does not make sense to create this class only to inject TranslationNormalizer

File: Processor/AttributeToFlatArrayProcessor.php [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    public function process($attribute)
+    {
+        $flatAttribute = array(
+                'type' => $attribute->getAttributeType(),
+                'code' => $attribute->getCode()
+            ) + $this->transNormalizer->normalize($attribute);
+
+        $flatAttribute = array_merge(
+            $flatAttribute,
+            [
```
Comment (Nuscly) [inline Processor/AttributeToFlatArrayProcessor.php]: It's normalizer job

File: Processor/AttributeToFlatArrayProcessor.php [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    {
+        $flatAttribute = [
+                'type' => $attribute->getAttributeType(),
+                'code' => $attribute->getCode()
+            ] + $this->transNormalizer->normalize($attribute);
+
+        $flatAttribute = array_merge(
+            $flatAttribute,
+            [
+                'group'                   => ($attribute->getGroup()) ? $attribute->getGroup()->getCode() : null,
```
Comment (Nuscly) [inline Processor/AttributeToFlatArrayProcessor.php:46]: It's normalizer job !

File: Processor/AttributeToFlatArrayProcessor.php [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+     *
+     * @return string
+     */
+    protected function getAttributeFamilyCodes(AttributeInterface $attribute)
+    {
+        $families = $attribute->getFamilies();
+
+        $familyCodes = [];
+        foreach ($families as $family) {
+            /** @var FamilyInterface $family */
```
Comment (BitOne) [inline Processor/AttributeToFlatArrayProcessor.php]: Nop, no IDE hint in the code please.

Comment (damien-carcel) [inline Processor/AttributeToFlatArrayProcessor.php]: Oups, forgot to remove this one, sorry !

**Pull request merged**